element.go

```diff
@@ -664,6 +664,11 @@ func (el *Element) Evaluate(opts *EvalOptions) (*proto.RuntimeRemoteObject, erro
 	return el.page.Context(el.ctx).Evaluate(opts.This(el.Object))
 }
 
+// Equal checks if the two elements are equal.
+func (el *Element) Equal(elm *Element) bool {
+	return el.MustEval(`elm => this === elm`, elm.Object).Bool()
```

Comment:
Please don't use Must inside non-Must methods. Check how other methods handle errors, such as `Element` and `MustElement`.

https://go-rod.github.io/#/error-handling